.travis.yml

```diff
@@ -26,11 +26,11 @@ stages:
 language: julia
 os:
   - linux
+  # - windows  # Unable to use Windows on Travis due to: https://github.com/JuliaTime/TimeZones.jl/pull/251#issuecomment-600360127
   - osx
-  - windows
 julia:
   - 1.0  # LTS
-  - 1.3  # Latest release
+  - 1.4  # Latest release
```

Comment:
Travis CI also supports `1` for the latest release. That is still using 1.3 at the moment though: https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/julia.rb#L185

Reply:
```suggestion
  - 1    # Latest release
```